app/js/controllers/main-ctrl.js

```diff
@@ -2,8 +2,139 @@
 
 angular
 .module('app.controllers')
-.controller('MainCtrl', function($scope, apiDescriptor) {
+.controller('MainCtrl', function($scope, apiDescriptor, Restangular) {
 	apiDescriptor.then(function(apiDescription) {
 		$scope.apidesc = apiDescription.data;
+		//console.log(preProcess.loadCurrentEBoard());
+		$scope.relatedSites = [
+			{	"name": "Tech@NYU",
+				"mainLink": "https://techatnyu.org/",
+				"prodLink": 'https://www.prodink.com',
+				"stagingLink": "https://www.staginglink.com"
+			},
+			{	"name": "Intranet",
+				"mainLink": "https://intranet.sexy/",
+				"prodLink": 'https://www.prodink.com',
+				"stagingLink": "https://www.staginglink.com"
+			},
+			{	"name": "Calendar",
+				"mainLink": "https://techatnyu.org/",
+				"prodLink": 'https://www.prodink.com',
+				"stagingLink": "https://www.staginglink.com"
+			},
+			{	"name": "RSVP",
+				"mainLink": "https://techatnyu.org/",
+				"prodLink": 'https://www.prodink.com',
+				"stagingLink": "https://www.staginglink.com"
+			},
+			{	"name": "Check-In",
+				"mainLink": "https://techatnyu.org/",
+				"prodLink": 'https://www.prodink.com',
+				"stagingLink": "https://www.staginglink.com"
+			},
+			{	"name": "Mailtrain",
+				"mainLink": "https://techatnyu.org/",
+				"prodLink": 'https://www.prodink.com',
+				"stagingLink": "https://www.staginglink.com"
+			},
+			{	"name": "Mosaico",
```

Comment:
you can remove the Mosaico block! it's integrated in Mailtrain now.

Reply:
Fixed